src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const declarations = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter(Boolean);
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve readability. Consider formatting the chained methods like the provided 'GOOD EXAMPLE' in the checklist.